api/v1beta1/glanceapi_types.go

```diff
@@ -85,6 +85,11 @@ type GlanceAPISpec struct {
 	// actual action pod gets started with sleep infinity
 	Debug GlanceAPIDebug `json:"debug,omitempty"`
 
+	// +kubebuilder:validation:Optional
+	// CephBackend - The ceph Backend structure with all the parameters
+	// TODO: -> implement a backend wrapper in a generic structure
```

Comment:
@fmount what is your though on the generic structure here. Is the JSON here somewhat free-form (could be anything) or is it something we could model in a Golang struct to verify it is in the right form ahead of time?

Reply:
Hi @dprince,
IMO in the current status this is free form: I left this TODO to brainstorm around a well defined wrapper where multiple backends can live within the same json structure.
It will be useful (and required) for multi-store support, and the same we'll see the same problem in cinder, but it can be part of a follow up work.